### DISCUSSION TITLE
Scout - Not showing trace log

### DISCUSSION THREAD

**Discussion by everseen83** [airsdk/Adobe-Runtime-Support#2007] [Q&A]
Hello :)

I can't see trace logs in new Scout app.
Also where can I find guidance or content on the issues below?
(Prefix : AIR-)

> AIR-5895: Scout - add query section to AIR Android manifests
 

##### Environment

- AIR SDK version: 33.1.1.889
- Development OS: Android
- Development IDE: VS code

**Comment by ajwfrost** [comment]
Hi

AIR-5895 was where we added a segment into the Android manifest files that AIR generates, in order to allow the AIR apps to access the new Scout process/service to get the configuration information from it. Plus we had to adjust the logic in ADT to allow this to then merge in any queries sections that were defined in the manifest additions from an AIR application descriptor..

In terms of this issue though.. is it just the trace log? Does the connection to Scout work i.e. you see the frame information appear?

If so, I suspect this may be more related to the implementation of "omit-trace-statements" that was added recently, where we do now actually take note of this value in the ASC2 compiler.. and if you've got a release build, this defaults to 'true'. So potentially you need to add a flag to the compilation to say `-omit-trace-statements=false`, or run it as a debug build?

thanks

**Reply by everseen83** [reply]
Q: Is it just the trace log? Does the connection to Scout work i.e. you see the frame information appear?
A : Yes, only the trace log is not visible, the rest are displayed normally.

However, "debug build" solved the problem.

Thanks for your help :)
